TestProgram.asm

```diff
@@ -771,15 +772,13 @@ ProgramRun_Loop:
 	Read_ADC_Channel(0)
 	lcall ConvertNum; converts voltage received to temperature
 	
-	Preheat_Abort(Mins_BCD,Temperature+1)
```

Comment:
This works for ours so we are going to leave it and go ahead with our solution that looks a lot like yours